### PR TITLE
Run Enmeshed tests on CI

### LINKED ISSUE #547: Run Enmeshed tests
See #537.

We should figure out passing secrets / environment variables.

We also should add some basic documentation about integrating other tests.

### PR DESCRIPTION
**Pull request by AlekSi**
Also bump deps, do small cleanups and port improvements from FerretDB.

Closes #547.

### CODE REVIEW

File: tests/enmeshed-runtime.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 runner: command
-dir: enmeshed-runtime
-args: ["sh", "-c", "npm i && CONNECTION_STRING=mongodb://localhost:27017 npx jest -i"]
+args: [../enmeshed-runtime.sh]
```
Comment (AlekSi) [inline tests/enmeshed-runtime.yml:3]: @b1ron I think we should document/promote that approach

Comment (b1ron) [reply tests/enmeshed-runtime.yml:3]: Nice, a much cleaner approach.

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    description: Issues that are not ready to be worked on; PRs that should skip CI
+
+  # https://hacktoberfest.com/participation/#spam
+  - name: spam
+    color: "#C5DEF5"
+    description: Spam issues and PRs
+
+  - name: trust
+    color: "#00FF00"
+    description: PRs that can access Actions secrets
```
Comment (b1ron) [inline .github/settings.yml:65]: The trust label was always a bit ambiguous to me, but now I know its intention.

Comment (b1ron) [reply .github/settings.yml:65]: :+1:

**Pull request merged**